src/App.tsx

```diff
@@ -11,48 +11,60 @@ import Home from "./modules/Home/components/Home/Home";
 //import ProtectedRoute from "./modules/Shared/components/ProtectedRoute/ProtectedRoute";
 import {
   createBrowserRouter,
-//  createHashRouter,
+  //  createHashRouter,
```

Comment:
It is not recommended to use hashrouter. You can find that in the documentation page :
![image](https://github.com/user-attachments/assets/43f26fac-869a-41b1-909a-901c052d87d0)

Currently of reload a page manually, it will give us page not found. This happens  because in single page application we only have the index.html and all pages are created by JavaScript. To fix this issue in deployment, we need to change the configuration to redirect all pages to index.html file.

For Netlify, we will need to create netlify.toml file in the top file directory and then add the below lines in  it.
```
[[redirects]]
from = "/*"
to = "/index.html"
status = 200
```

Reply:
Done